src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
+  let currentState = { ...state };
```

Comment:
The name `currentState` for the object copy is good, but it would be even clearer if it was named `stateClone` or `newState` to indicate that this is a new version of the state after applying each action.